sdk/src/connections/mipi/dragonboard/target_definitions.h

```diff
@@ -32,6 +32,7 @@
 #ifndef TARGET_DEFINITIONS_H
 #define TARGET_DEFINITIONS_H
 
+static const char *EEPROM_NAME = "24c1024";
```

Comment:
This define is duplicated in each `target_definitions.h` of each platform. Maybe we should consider creating a `common_target_definitions.h` header as this definition is not platform-dependent.

Reply:
I would keep this as it is. This way, we have everything related to one target in the same place. If the list of duplicated things largely increases then yes, it would make sense to move them to some common header.